ng-dev/auth/shared/firebase.ts

```diff
@@ -17,6 +19,24 @@ export async function loginToFirebase() {
   /** The type of OAuth dance to do based on whether a session display is available. */
   const oAuthDance = process.env.DISPLAY ? authorizationCodeOAuthDance : deviceCodeOAuthDance;
   try {
+    // Only present intial information about usage of login when it appears that the user has
+    // not logged into the service in the past.
+    if (!(await hasEvidenceOfPreviousLogin())) {
+      Log.warn(Array(80).fill('#').join(''));
+      Log.warn('The ng-dev auth service uses Google OAuth credentials to log in and create a');
+      Log.warn('short lived credential used for authenticating with the ng-dev service.');
+      Log.warn('');
+      Log.warn('In addition to logging in using Google credentials, upon first login you will be');
+      Log.warn('prompted to associate your Github account to your login, allowing the service to');
+      Log.warn('perform requests on your your behalf.');
```

Comment:
Nit: You may want to use `Log.info` for messages that should not be yellow. Just in case you want to make some phrases more prominent.

Reply:
I wanted this to stand out since its the explanation of what is going to happen on login. Do you think it shouldn't be?

Reply:
I just wanted to see if you just wanted to make a subet of these lines yellow. Now all are yellow

Reply:
I think having the whole block yellow is good in this specific case.

Reply:
sgtm!